build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+plugins {
+    id("com.android.application") version "8.1.2" apply false
+    id("com.github.spotbugs") version "5.1.3" apply false
+}
+
+allprojects {
+    tasks.withType<JavaCompile> {
+        options.compilerArgs.add("-Xlint:deprecation")
+    }
+}
```

Comment:
The `clean` task no longer seems to exist. Is this on purpose?

Reply:
Yes. This has been dropped in new templates which I assume means is no longer required. The clean task still works fine without it.